src/databricks/labs/ucx/install.py

```diff
@@ -245,6 +240,12 @@ def _prompt_for_new_installation(self) -> WorkspaceConfig:
         configure_groups = ConfigureGroups(self.prompts)
         configure_groups.run()
         include_databases = self._select_databases()
+
+        # Checking if the user wants to define a default owner group.
+        default_owner_group = None
```

Comment:
What happens later when there is no default owner group? Does the default owner group if specified take priority over the logic we implement to find the owner (group)?